src/styles/colour/index.md.njk

```diff
@@ -0,0 +1,19 @@
+---
+title: Colour
+description:
+section: Styles
+aliases:
+layout: layout-styles.njk
+---
+
+{% from "_example.njk" import example %}
+
+Always use the GOV.UK colour palette.
+
+## Colour contrast
+The colour contrast ratio for text and interactive elements should be at least 4.5:1 as recommended by the W3C. Test your service to meet this standard.
```

Comment:
Can we change to: 'You must make sure the colour contrast ratio for text and interactive elements in your service is at least 4.5:1 as recommended by the W3C.'